### DISCUSSION TITLE
I want to deploy my SSR app to CloudWays hosting

### DISCUSSION THREAD

**Discussion by hassam-saeed** [vikejs/vike#1413] [Discussion]
I want to convert my [project](https://github.com/DevAxonOfficial/SSR-Frontend) to [CloudWays](https://www.cloudways.com/en/?id=863506&data1=AT1&data2=AT2) but in the docs nothing is mentioned, Anyone please help me?
It is already deployed on Vercel but the client has a requirement to deploy it on cloudways.

**Comment by brillout** [comment]
https://brillout.github.io/rules#try
